src/site/content/en/blog/trusted-types/index.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Trusted Types
+subhead: Use Trusted Types to prevent DOM XSS vulnerabilities.
+authors:
+  - koto
+date: 2020-03-25
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+hero: hero.png
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+# thumbnail: thumbnail.png
+alt: Code snippets demonstrating Cross-Site Scripting vulnerabilities.
+description: |
+  Introducing Trusted Types - a browser API to prevent DOM-Based Cross
+  Site Scripting in modern web applications.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - security
+  - trusted-types
+---
+
+## Why should you care?
+
+DOM-Based Cross-Site Scripting (DOM XSS) is one of the most common web
+application security vulnerabilities, and it's very easy to introduce it in
+your application. [Trusted Types](https://github.com/w3c/webappsec-trusted-types)
+give you the tools to write, security
+review and maintain DOM-XSS free applications by making the dangerous Web API
+functions secure by default. Trusted Types are supported in Chrome 82, with
+a [polyfill](https://github.com/w3c/webappsec-trusted-types#polyfill) available
+for other browsers.
+
+{% Aside 'key-term' %}
+DOM Based Cross-Site Scripting (DOM XSS) happens when data from user controlled
+_source_ (like user name, or redirect URL taken from the URL fragment)
+reaches a _sink_ - function that can execute arbitrary JavaScript code
+(like `eval` or `.innerHTML` property setter).
+{% endAside %}
+
+## Chrome launches Trusted Types
+
+For many years [Cross Site Scripting](https://owasp.org/www-community/attacks/xss/),
+or XSS is one of the most prevalent—and dangerous—web security vulnerabilities.
+
+There are two distinct groups of Cross Site Scripting. Some
+XSSes are caused by the server-side code that insecurely creates the HTML code
+forming the website. Others have a root cause on the client, where the JavaScript
+code calls dangerous functions with user-controlled content.
+
+To prevent server-side XSS, don't generate
+HTML by concatenating strings and use safe contextual-autoescaping templating
+libraries instead. Use a [nonce-based Content Security Policy](https://csp.withgoogle.com/docs/strict-csp.html) for additional mitigation against the bugs as they inevitably happen.
```

Comment:
I generally agree with Joe but am OK with this paragraph, because it doesn't add a huge amount of extraneous information to the content. I do agree that we should link to an article that guides users through solving server-side XSS.

Reply:
Added a link, I'd prefer if the paragraph was kept, as it defines more clearly what this variant of XSS is about, as opposed to server side XSSes.